src/critics/smoother_critic.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 FastSense, Samsung Research
+#include "mppic/critics/smoother_critic.hpp"
+
+namespace mppi::critics
+{
+
+void SmootherCritic::initialize()
+{
+  auto getParam = parameters_handler_->getParamGetter(name_);
+
+  getParam(vx_power_, "smoother_vx_cost_power", 1);
+  getParam(vy_power_, "smoother_vy_cost_power", 1);
+  getParam(wz_power_, "smoother_wz_cost_power", 1);
+  getParam(vy_weight_, "smoother_vx_cost_weight", 2.0);
+  getParam(vx_weight_, "smoother_vy_cost_weight", 2.0);
+  getParam(wz_weight_, "smoother_wz_cost_weight", 2.0);
+}
+
+void SmootherCritic::score(
+  const geometry_msgs::msg::PoseStamped & /* robot_pose */,
+  const models::State & state,
+  const xt::xtensor<double, 3> & /* trajectories */,
+  const xt::xtensor<double, 2> & /* path */,
+  xt::xtensor<double, 1> & costs,
+  nav2_core::GoalChecker * /*goal_checker*/)
+{
+  using namespace xt::placeholders;   // NOLINT
+  xt::xtensor<double, 1> cost;
+
+  auto vx = state.getControlVelocitiesVX();
+  auto vy = state.getControlVelocitiesVY();
```

Comment:
What if there is no V velocity (e.g. non holonomic)?

Reply:
removed this critic from pr